wp-content/themes/chrisdinh-photography/app/View/Composers/PageGallery.php

```diff
@@ -14,13 +14,24 @@ class PageGallery extends Composer {
     ];
 
     private function getGalleryItems() {
-        $posts = get_posts([
-            'post_type' => 'gallery_items',
-            'post_status' => 'publish',
-            'posts_per_page' => -1
-        ]);
+        $allItems = get_field('gallery__gallery-images');
```

Comment:
If $allItems can be null or empty, consider adding a guard clause at the start of the function to return early if no items are available.

Reply:
I would argue that we don't need to because if the field type of `gallery__gallery-images` returns an array (based on the field return value) and will be empty if no items are added in the admin - but for future proofing you're right there should be a conditional there. Added in [this commit](https://github.com/jonathanmdinh/itschrisdinh.com/pull/6/commits/0d2ad70310bd0bbdfec5a4af2349412acd7ace9c)

Reply:
Also based on that guard conditional, an empty `$items` array is returned by default